clients/java/src/main/java/io/zeebe/client/api/response/DeployWorkflowResponse.java

```diff
@@ -17,7 +17,7 @@
 
 import java.util.List;
 
-public interface DeploymentEvent {
+public interface DeployWorkflowResponse {
```

Comment:
The name `DeployWorkflowResponse` does not feel right for me. The deployment command can contain multiple resources and the command does not contain the term `workflow`. 

```
DeployWorkflowResponse deployWorkflowResponse =
          client.newDeployCommand().addResourceFromClasspath("demoProcess.bpmn").send().join();
```

What do you think about `DeploymentResponse` or `DeployResponse`?

Reply:
Will roll it back